src/azure-cli/azure/cli/command_modules/backup/_help.py

```diff
@@ -35,6 +35,14 @@
     crafted: true
 """
 
+helps['backup container unregister'] = """
+type: command
+short-summary: Unregister a Backup Container to make the underlying 'resource' be protected by another vault.
+examples:
+  - name: This command allows Azure Backup to unregister a ‘Backup Container’ so that the underlying 'resource' can be protected to another vault, if required. The existing backup data in this vault should be deleted before 'unregister' can be performed.
```

Comment:
It will be better to give a brief but accurate example description. In addition, 'Backup Container’ is strange which may make document generation wrong.